execution/gethexec/sequencer.go

```diff
@@ -1458,7 +1458,12 @@ func (s *Sequencer) StopAndWait() {
 			wg.Add(1)
 			go func() {
 				defer wg.Done()
-				err := forwarder.PublishTransaction(item.ctx, item.tx, item.options)
+				var err error
+				if source == "timeboostAuctionResolutionTxQueue" {
```

Comment:
if source is used by logic and not just for logging - it needs to be an enum and not a string.